clang/lib/CodeGen/ABIInfoImpl.cpp

```diff
@@ -296,10 +296,16 @@ bool CodeGen::isEmptyRecord(ASTContext &Context, QualType T, bool AllowArrays,
     return false;
 
   // If this is a C++ record, check the bases first.
-  if (const CXXRecordDecl *CXXRD = dyn_cast<CXXRecordDecl>(RD))
+  if (const CXXRecordDecl *CXXRD = dyn_cast<CXXRecordDecl>(RD)) {
```

Comment:
```suggestion
  if (const auto *CXXRD = dyn_cast<CXXRecordDecl>(RD)) {
```
Since we're already touching the line...